### PR TITLE
Fix options switch alignment within settings

### PR DESCRIPTION
**Pull request by jarolrod**
Fixes alignment of the option switch within a settings row.

| master | PR |
| --------- | ---- |
| master-option | fix-option-pr |

Closes bitcoin-core/gui-qml#159

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/161)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/161)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/161)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/161)

### CODE REVIEW

**Comment by hebasto** [comment]
cc @GBKS

**Comment by GBKS** [comment]
That looks correctly centered now. Is this automatic centering or by setting pixel coordinates (or top padding)? If it's the latter, then it may not be centered anymore if the content on the left changes (like a translation being longer).

**Comment by jarolrod** [comment]
@GBKS the centering is automatic, it's handled by the RowLayout. This change constrains the OptionSwitch to only the space needed for it's visual elements instead of having padding all around.

**Comment by hebasto** [comment]
`root.width` == `indicator.implicitWidth` -- is it coincidence? If not, why a constant is duplicated?

**Comment by jarolrod** [comment]
updated from 250bb9f to 5af9c73

changes: set indicators implicitWidth/height to parent width/height

File: src/qml/controls/OptionSwitch.qml [hunk 1]
```diff
@@ -7,9 +7,12 @@ import QtQuick.Controls 2.15
 
 Switch {
     id: root
+    padding:0
```
Comment (shaavan) [inline src/qml/controls/OptionSwitch.qml:10]: nit:

```suggestion
    padding: 0
```

**Review by shaavan — COMMENTED** [review]
ACK modulo nit

- The optionswitch are correctly aligned.

Here's a screenshot I took on Ubuntu 22.04 with Qt version 5.15.3

**Screenshot:**
|Master|PR|
|---------|----|
|Screenshot from 2022-09-07 00-36-42|Screenshot from 2022-09-07 00-28-21|

**Comment by jarolrod** [comment]
updated from [5af9c73 ](https://github.com/bitcoin-core/gui-qml/commit/5af9c73e46f28b5b71ef43c1ea8d269c18a82ff8)to [e6f8728](https://github.com/bitcoin-core/gui-qml/commit/e6f87282d39ee5a86aa2e1835280097e2843bb7e) ([pr161.01](https://github.com/jarolrod/gui-qml/tree/pr161.01) -> [pr161.02](https://github.com/jarolrod/gui-qml/tree/pr161.02), [compare](https://github.com/jarolrod/gui-qml/compare/pr161.01..pr161.02))

Changes: addressed spacing nit

File: src/qml/controls/OptionSwitch.qml (hunk 1 above)
Comment (jarolrod) [reply src/qml/controls/OptionSwitch.qml]: done

**Comment by hebasto** [comment]
https://github.com/bitcoin-core/gui-qml/pull/169#pullrequestreview-1101569629:

> Confirmed that this fixes the issue, and is a better approach than 161.

**Pull request closed without merging**